src/qt/transactiondesc.cpp

```diff
@@ -36,13 +36,41 @@ QString TransactionDesc::FormatTxStatus(const interfaces::WalletTxStatus& status
 {
     int depth = status.depth_in_main_chain;
     if (depth < 0) {
+        /*: Text explaining the current status of a transaction, shown in the
+            status field of the details window for this transaction. This status
+            represents an unconfirmed transaction that conflicts with another
```

Comment:
"another" here is a little bit contradictory as it suggests that the other transaction is also unconfirmed; propose s/another/a/